geom.go

```diff
@@ -162,6 +162,25 @@ func (g *GeoM) Rotate(theta float64) {
 	g.ty = ty
 }
 
+// Skew the image.
+func (g *GeoM) Skew(skewX, skewY float64) {
+
+	sx64 := math.Tan(skewX)
+	sy64 := math.Tan(skewY)
+	sx, sy := float32(sx64), float32(sy64)
+
+	a := g.c * sy
+	b := (g.d_1 + 1) * sy
+	c := (g.a_1 + 1) * sx
+	d := g.b * sx
```

Comment:
Is this calculation really correct? I expect this multiplying:

```
| 0          tan(skewX) 0 || a b tx |
| tan(skewY) 0          0 || c d ty |
| 0          0          1 || 0 0 1  |
```

Then tx and ty are also affected?

Reply:
Sorry but

```
| 1          tan(skewX) 0 || a b tx |
| tan(skewY) 1          0 || c d ty |
| 0          0          1 || 0 0 1  |
```

is the correct calculation I think. (https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/geom/Matrix.html includes a wrong skew matrix?)

Reply:
You're absolutely right; I thought that result looked fishy! Thanks for the catch, and sorry for my confusion. I'll fix this today.